### ISSUE TITLE
query for my use case

### ISSUE THREAD

**Issue by lakshy-amagi** [contribsys/faktory_worker_go#79]
I want to confirm here can use faktory_worker for my use case. 

My use case 
Let's suppose 
we have n customers c1,c2,....,cn
we have m updates for c1 and k updates for c2
All the updates come at the same time

I want to have this workflow in place
I need one worker per customer and all the updates for the respective customer should be synchronous. But updates for multiple customers can be asynchronous.

So something like
w1 - worker for customer c1
w2 - worker for customer c2

[    c1    ]            [    c2    ]      ........      [   cn   ]
[ u1,u2,...um]     [u1,u2,...uk]   

c1 and c2 can be updated in parallel but updates for each customer should be synchronous.
Can I achieve this case? If yes can you please guide me on how?
If no, is there any alternative for it?

**Comment by mperham** [comment]
This is out of scope for Faktory support. I can answer specific questions about Faktory, I can’t architect your app for you.

**Comment by lakshy-amagi** [comment]
> This is out of scope for Faktory support. I can answer specific questions about Faktory, I can’t architect your app for you.

I was asking can we do it with faktory. Sorry for framing my question wrongly.
